src/test/java/teammates/test/driver/TestStringHelper.java

```diff
@@ -0,0 +1,56 @@
+package teammates.test.driver;
+
+import teammates.common.util.Assumption;
+import teammates.common.util.FieldValidator;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.StringHelper;
+
+/**
+ *String Helper class for tests.
```

Comment:
Missing space. Also, `Holds String-related helper functions used only in tests.` might be better?